db/migrate/20250619163228_create_potential_duplicate_persons.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class CreatePotentialDuplicatePersons < ActiveRecord::Migration[7.2]
+  def change
+    create_table :potential_duplicate_persons do |t|
+      t.references :duplicate_checker_job_run, null: false, foreign_key: { to_table: :duplicate_checker_job_runs }
+      t.references :original_user, type: :integer, null: false, foreign_key: { to_table: :users }
+      t.references :duplicate_person, type: :integer, null: false, foreign_key: { to_table: :persons }
+      t.string :algorithm, null: false
```

Comment:
This might be misleading for other devs trying to understand your code, because this `algorithm` only affects the name string comparison. But we also compare DOB (and nationality??) which affects the resulting `score`, but is completely unchanged by `algorithm`.
Perhaps something like `name_matching_algorithm` would be more exact? It's longer but it's also more accurate about what actually happens in the code